### PR TITLE
Mirror environment setup from official Rust containers

### PR DESCRIPTION
**Pull request by mbestavros**
And remove the sanity check.

Environment variables come from [here](https://github.com/rust-lang/docker-rust/blob/master/Dockerfile-debian.template). They influence where `rustup` installs things, according to its [README](https://github.com/rust-lang/rustup#choosing-where-to-install).

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,13 +1,13 @@
 FROM quay.io/enarx/fedora
 
-ENV PATH /root/.cargo/bin:$PATH
+ENV RUSTUP_HOME=/usr/local/rustup \
+    CARGO_HOME=/usr/local/cargo \
+    PATH=/usr/local/cargo/bin:$PATH
```
Comment (npmccallum) [inline Dockerfile]: I'm confused by this. The `rustup` install script doesn't install things to `/usr/local` but rather to `/root/.cargo`. So I'm not sure how these paths will help...

Comment (npmccallum) [reply Dockerfile]: What is the dockerfile you got this from?

Comment (mbestavros) [reply Dockerfile]: `RUSTUP_HOME` and `CARGO_HOME` influence where `rustup` installs everything, according to its [README](https://github.com/rust-lang/rustup#choosing-where-to-install), as long as they're defined before running `rustup`.

Comment (npmccallum) [reply Dockerfile]: I suspect this is supposed to be:

```
ENV RUSTUP_HOME=/root/.rustup \
    CARGO_HOME=/root/.cargo \
    PATH=/root/.cargo/bin:$PATH
```

**Pull request merged**